### ISSUE TITLE
Функционал формы

### ISSUE THREAD

**Issue by adubovskoy** [dru-io/promo#8]
В промо-странице внизу есть форма.

Промо-сайт скорее всего будет отдельным drupal-сайтом или вообще статикой.

Предложение: в профиле пользователя dru.io сделать галочку "хочу получать вопросы с промо-страницы". 

Варианты дальше:

1. Форма отправляет на dru.io тело письма, далее dru.io отправляет всем у кого галочка письмо . Плюсы: просто. Минусы: много почты, это проблемы, вопросы с доставкой, быстро отправить 1000 писем - это еще вопросы со спамом (или подключением чего-то типа amazon ses). 
2. Форма отправляет на dru.io id заполенния формы, само тело письма - в открытом доступе, далее dru.io делает сущность нотифи, которая мигает в звоночке наверху сайта, там просто ссылка на результаты формы промо. Без почты. 
3. В gitter-чат слать уведомления со ссылкой.

**Comment by politsin** [comment]
Такая форма будет причиой для многих кампаний не давать ссылку на этот сайт со словами "вот тут можете почитать про друпал"

**Comment by politsin** [comment]
А для отправивших людей будет странно получить 20+ обращений от сейлов компаний. Ну и всё потом придёт к грызне по поводу кто будет получать эти письма а кто не будет.

**Comment by ArmGono** [comment]
Тоже считаю, что форма излишне. Может вместо формы будет корректно показывать студии, которые работают с друпалом?

**Comment by Niklan** [comment]
@politsin здравая мысль. Действительно может отпугнуть.
@ArmGono а выдавая одни студии может как-то задеть фрилансеров, например.

**Comment by ArmGono** [comment]
@Niklan согласен. А может для фрилансеров ссылку на раздел dru.io по поиску исполнителей?

**Comment by adubovskoy** [comment]
Может форму показывать по аргументу в урле? ?form=default, если есть, показывать. По умолчанию - прятать. С сайтов сообществ/блогов давать ссылочку с формы, в личных письмах если кто отправляет - без.

**Comment by ArmGono** [comment]
Тут опять встанет вопрос, если давать ссылку с формой, кто получит письма?

**Comment by adubovskoy** [comment]
> Тут опять встанет вопрос, если давать ссылку с формой, кто получит письма?

Все кто включил у себя галочку в профиле. А можно вообще все показывать анонимам и в документации указать ссылку. Или 10 последних, вью сделать.

**Comment by politsin** [comment]
Есть классные техники принятия решений из разряда:
Если не удастся придти к одному мнению за 5 реплик или до *** (допустим вечера вторника)  - признаём вопрос сложным и не делаем в первой итерации

**Comment by ArmGono** [comment]
> Все кто включил у себя галочку в профиле. А можно вообще все показывать анонимам и в документации указать ссылку. Или 10 последних, вью сделать.

В принципе лично я  проблемы не вижу, но все же опять возвращаемся к следующему пункту:

> А для отправивших людей будет странно получить 20+ обращений от сейлов компаний. Ну и всё потом придёт к грызне по поводу кто будет получать эти письма а кто не будет.

В этом случае может помочь подсказка к форме, о том, что письмо получат разные разработчики Друпала и.т.п. чтобы человек, который отправляет форму осознано делал "подписку" на письма с предложениями от разработчиков. Думаю в данном случае негатив сильно уменьшится.

**Comment by politsin** [comment]
Можно постить сообщение без контактных данных клиента в раздел "заказы", а человеку послать ссылку на его сообщение.
Все загеристированные могут там "ответить", а все ответы сопровождаются оставленными в профиле комментариями.

**Comment by adubovskoy** [comment]
>Друпала и.т.п. чтобы человек, который отправляет форму осознано делал "подписку" на письма с предложениями от разработчиков. Думаю в данном случае негатив сильно уменьшится.

Имхо как раз и нет такого механизма, который позволит получить сразу много предложений от компаний. 

>Можно постить сообщение без контактных данных клиента в раздел "заказы", а человеку послать ссылку на его сообщение.

Да, может не сразу, но прикрутить надо, согласен. Вопрос в том что заказ надо тогда генерировать автоматически от робота, в стиле "Нами получен запрос с формы ... заказчик не оставил подробного описания, требуется помощь с ТЗ и консультирование".

**Comment by pselfin** [comment]
А может  отправлять на регистрацию на друио?
Там же форма не "Объявить тендер", а "Возникли вопросы"?
На дру ио уже хочет пусть вопросы задает, или публикует заказ... 
Мол вот сайт сообщества, и делай там что тебе требуется.
зы а разрабы кто уже захочет на своих ресурсах будет эту страничку копировать и свою форму вешать, какую надо.

**Comment by byteval** [comment]
Форму можно заменить на найти специалиста/студию/разместить заказ, с автопубликацией на dru.io. 
Либо просто текстом Как найти исполнителя для сайта на Друпал.

**Comment by Alex-Malkov** [comment]
По ссылке http://drupal.dru.io/thanks остался пункт меню "Задать вопрос".

**Comment by adubovskoy** [comment]
>По ссылке http://drupal.dru.io/thanks остался пункт меню "Задать вопрос".

убран
